server/services/query-memory-db/queries/business-process-rejection-reasons.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+var ensureObject   = require('es5-ext/object/valid-object')
+  , ensureString   = require('es5-ext/object/validate-stringifiable-value')
+  , ensureDatabase = require('dbjs/valid-dbjs')
+  , rejectionReason;
+
+rejectionReason = function (ownerType, types, value) {
+	var reason = {};
+	reason.types = typeof types === 'object' ? types : [];
+	reason.value = value || '';
+	reason.ownerType = ownerType || '';
+
+	return reason;
+};
+
+module.exports = exports = function (db) {
+	ensureDatabase(db);
+	var otherValue, result = {
+		hasOnlySystemicReasons: true,
+		rejectionType: null,
+		rejectionReasons: [],
+		service: {
+			type: null,
+			id: null,
+			businessName: null
+		},
+		operator: {
+			id: null,
+			name: null
+		},
+		processingStep: {
+			path: null,
+			label: null
+		},
+		date: null
+	};
+
+	var setProcessingStepData = function (step) {
+		result.processingStep.label = step.label;
```

Comment:
Let's pass result directly to function, so there is no doubt as to what we're talking about here